src/mermaidAPI.js

```diff
@@ -678,7 +678,7 @@ export default mermaidAPI;
  *     flowchart:{
  *       diagramPadding:8,
  *       htmlLabels:true,
- *       curve:'linear',
+ *       curve:'basis',
```

Comment:
This wrong default value is fixed.

Besides, shall I complete the missing default values here such as `pie`?

Reply:
That would be great!